jumpscale/clients/stellar/stellar.py

```diff
@@ -800,7 +802,7 @@ def check_is_payment_transaction(self, transaction_hash):
         results = response["_embedded"]["records"][0]
         return results["type"] == "payment"
 
-    def get_asset(self, code="TFT", issuer=None) -> stellar_sdk.Asset:
+    def _get_asset(self, code="TFT", issuer=None) -> stellar_sdk.Asset:
```

Comment:
Hiding this function breaks stellar and automation tests https://github.com/threefoldtech/js-sdk/search?q=get_asset

Reply:
Good catch, adapted both now. When having a wallet with sdk abstraction, there should be no need to use the sdk types.

Reply:
If you could run black on the changed files too would be great, to make the ci pass

Reply:
Thank you @waleedhammam  and sorry I did not do that before, automated black formatting in my editor was broken.